src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1216,6 +1216,8 @@ trait Definitions extends api.StandardDefinitions {
     lazy val Object_isInstanceOf = newT1NilaryMethod(ObjectClass, nme.isInstanceOf_Ob, FINAL | SYNTHETIC | ARTIFACT)(_ => BooleanTpe)
     lazy val Object_asInstanceOf = newT1NilaryMethod(ObjectClass, nme.asInstanceOf_Ob, FINAL | SYNTHETIC | ARTIFACT)(_.typeConstructor)
     lazy val Object_synchronized = enterNewT1Method(ObjectClass, nme.synchronized_, FINAL)(_.typeConstructor)
+    // used to generate scaladoc from AnyRef.scala
+    lazy val AnyRef_synchronized = enterNewT1Method(AnyRefClass, nme.synchronized_, FINAL)(_.typeConstructor)
```

Comment:
This looks rather strange, `AnyRef` is defined as a type alias, not sure what entering a method means. In any case, looking up `synchronized` already gives the `Object_synchronized` symbol

```
scala> AnyRefClass.tpe.member(TermName("synchronized")) eq Object_synchronized
val res4: Boolean = true
```

Reply:
Digging in a bit more deeper, I found 4449c5af3 from 2011. When I comment that out, `synchronized` seems to grab the Scaladoc, so I added 48c9245 to save the modified `Object#synchronized` symbol as a `val` so I can convert it back to the original `Object_synchronized`.